site/en/docs/extensions/reference/permissions/index.md

```diff
@@ -82,25 +84,25 @@ Most Chrome extension permissions can be specified as optional, with the followi
     </tr>
     <tr id="mdns">
       <td><code>"mdns"</code></td>
-      <td>Grants the extension access to the <a href="mdns">chrome.mdns</a> API.</td>
+      <td>Grants the extension access to the <a href="../mdns">chrome.mdns</a> API.</td>
     </tr>
     <tr id="proxy">
       <td><code>"proxy"</code></td>
-      <td>Grants the extension access to the <a href="mdns">chrome.proxy</a> API to manage Chrome's
-        proxy settings.</td>
+      <td>Grants the extension access to the <a href="../proxy">chrome.proxy</a> API to manage
+        Chrome's proxy settings.</td>
     </tr>
     <tr id="tts">
       <td><code>"tts"</code></td>
-      <td>The <a href="tts">chrome.tts</a> API plays synthesized text-to-speech (TTS).</td>
+      <td>The <a href="../tts">chrome.tts</a> API plays synthesized text-to-speech (TTS).</td>
     </tr>
     <tr id="ttsEngine">
       <td><code>"ttsEngine"</code></td>
-      <td>The <a href="ttsEngine">chrome.ttsEngine</a> API implements a text-to-speech(TTS) engine
-        using an extension.</td>
+      <td>The <a href="../ttsEngine">chrome.ttsEngine</a> API implements a text-to-speech (TTS)
+        engine using an extension.</td>
     </tr>
     <tr id="wallpaper">
       <td><code>"wallpaper"</code></td>
-      <td><strong>ChromeOS only</strong>. Use the <a href="ttsEngine">chrome.wallpaper</a> API
+      <td><strong>ChromeOS only</strong>. Use the <a href="../wallpaper">chrome.wallpaper</a> API
```

Comment:
```suggestion
      <td><strong>ChromeOS only</strong>. Use the
      <a href="/docs/extensions/reference/wallpaper/">chrome.wallpaper</a> API
```

Generally we've prefered using the full absolute path in the extensions docs so that we can use find-replace to update all references in one go.